src/pages/Search/components/NotFoundPage/NotFoundPage.module.scss

```
@@ -0,0 +1,38 @@
+@use "src/utils/styles/mediaQuery.scss" as media;
+
+.not-found-page__container {
+  padding-top: 180px;
+  max-width: 980px;
+  margin: 0 auto;
+}
+
+.not-found-page__container__title {
+  width: 544px;
+  height: 100px;
+  color: #000;
+  font-family: SUIT, system-ui;
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: normal;
+}
```

```suggestion
.not-found-page__container {
  padding-top: 180px;
  max-width: 980px;
  margin: 0 auto;

&__title {
  width: 544px;
  height: 100px;
  color: #000;
  font-family: SUIT, system-ui;
  font-size: 40px;
  font-style: normal;
  font-weight: 400;
  line-height: normal;
}
}
```
bem 구조를 활용하는 방식으로 변경해주세요

---

넵 수정했습니다!

3581d11e9de93625d8ffbb25b15f417a28c340bc